source/contributed/tutorialMonkeyPatching.md

```diff
@@ -0,0 +1,210 @@
+# Monkey Patching Tutorial for Screeps
```

Comment:
I disagree with referring to this as monkey patching. Monkey patching has more to do with modifying object instances than modifying prototypes. I suggest renaming this article to "Modifying Object Prototypes", and change all article statements accordingly.